testkit/src/main/scala/org/bitcoins/testkit/core/gen/TLVGen.scala

```diff
@@ -1,7 +1,11 @@
 package org.bitcoins.testkit.core.gen
 
 import org.bitcoins.commons.jsonmodels.dlc.DLCFundingInputP2WPKHV0
-import org.bitcoins.commons.jsonmodels.dlc.DLCMessage.{DLCAccept, DLCOffer}
+import org.bitcoins.commons.jsonmodels.dlc.DLCMessage.{
```

Comment:
Need TLVGen for new oracle and contract infos

Reply:
ping

Reply:
(This will be a subsequent PR, just documenting so that I can make a list of things to do in subsequent PRs more easily)